techniques/failures/F48.html

```diff
@@ -140,21 +127,19 @@ <h3>Example 2: Election results displayed using preformatted text</h3>
       </section>
    </section>
 
-<section id="related"><h2>Related Techniques</h2><ul>
+<section id="related"><h2>関連テクニック</h2><ul>
       <li><a href="../html/H51">H51: Using table markup to present tabular information</a></li>
    </ul></section>
-<section id="tests"><h2>Tests</h2>
-      <section class="procedure" id="procedure"><h3>Procedure</h3>
+<section id="tests"><h2>検証</h2>
+      <section class="procedure" id="procedure"><h3>手順</h3>
          <ol>
-            <li>Check to see if the pre element is used</li>
-            <li>For each occurrence of the pre element, check
-                                    whether the enclosed information is tabular.</li>
+            <li>pre 要素が使われているかどうかをチェックする。</li>
+            <li>pre 要素が出現するたびに、囲まれた情報が表形式であるかどうかを確認する。</li>
```

Comment:
check の訳を他のテクニックと合わせました。
```suggestion
            <li>pre 要素が出現するたびに、囲まれた情報が表形式であるかどうかをチェックする。</li>
```